src/components/SlateEditor/plugins/uuDisclaimer/DisclaimerForm.tsx

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { FieldProps, Form, Formik, FormikValues } from "formik";
+import { Dispatch, SetStateAction, useMemo } from "react";
+import { useTranslation } from "react-i18next";
+import { Descendant } from "slate";
+import styled from "@emotion/styled";
+import { ButtonV2 } from "@ndla/button";
+import { colors, misc, spacing } from "@ndla/core";
+import { ModalBody } from "@ndla/modal";
+import SafeLink from "@ndla/safelink";
+import { UuDisclaimerEmbedData } from "@ndla/types-embed";
+import { Text } from "@ndla/typography";
+import { plainTextToEditorValue } from "../../../../util/articleContentConverter";
+import FormikField from "../../../FormikField";
+import validateFormik, { RulesType } from "../../../formikValidationSchema";
+import PlainTextEditor from "../../PlainTextEditor";
+
+const DISCLAIMER_EXAMPLES_LINK =
+  "https://docs.google.com/spreadsheets/d/1g8cCqgS4BvaChHX4R6VR5V5Q83fvYcMrgneBJMkLWYs/edit?usp=sharing";
```

Comment:
Antar dette er kun for utvikling

Reply:
Veit ikke enda; har brukt lenka som ligger i Figma-boardet. Får sjekke det med Hedvig.

Reply:
Lenka i figma er jo berre for å ha eit eller anna å lenke til.

Reply:
Var visst ikkje det likevel...